shared/trainer/defaults.yaml

```diff
@@ -32,14 +32,14 @@ data:
       p: 0.5
     - name: resize
       ignore: false
-      new_size: 256
+      new_size: 640
     - name: crop
-      ignore: false
-      height: 224
-      width: 224
+      ignore: true
+      height: 640
+      width: 640
```

Comment:
We should put maybe 560 (aka the same ratio than 224/256) for the crop size even tho we're ignoring it right now?

Reply:
Why not